amundsen_application/static/js/ducks/tableMetadata/api/v0.ts

```diff
@@ -34,37 +34,53 @@ export type DescriptionAPI = { description: string } & MessageAPI;
 export type LastIndexedAPI = { timestamp: string } & MessageAPI;
 export type PreviewDataAPI = { previewData: PreviewData } & MessageAPI;
 export type TableDataAPI = { tableData: TableData } & MessageAPI;
-export type RelatedDashboardDataAPI = { dashboards: DashboardResource[] };
+export type RelatedDashboardDataAPI = {
+  dashboards: DashboardResource[];
+} & MessageAPI;
 
 export function getTableData(
```

Comment:
We reverted to having a `getTableData` call that just gets table metadata, and now also have a `getTableDashboards` call.